src/App.js

```diff
@@ -1,24 +1,18 @@
-import logo from './logo.svg';
+import { Route, Switch } from 'react-router-dom';
+import Header from './Components/Header/Header.js';
+import RGB from './Components/RGB/RGB.js';
+import NotFound from './Components/NotFound/NotFound.js';
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <main>
+      <Header />
+      <Switch>
+        <Route path="/rgb/:r/:g/:b" component={RGB} />
```

Comment:
i'm thinking you should have a route that redirects from the home page to avoid that 404 issue:
```
<Route path="/">
  <Redirect to="/rgb/whatevervalues" />
</Route>
```

Reply:
currently you're not handling someone going to your initial page with a route so it defaults to the notfound component